plugins/gcp/resoto_plugin_gcp/resources/sqladmin.py

```diff
@@ -578,7 +578,7 @@ class GcpSqlDatabaseInstance(GcpResource):
     secondary_gce_zone: Optional[str] = field(default=None)
     server_ca_cert: Optional[GcpSqlSslCert] = field(default=None)
     service_account_email_address: Optional[str] = field(default=None)
-    settings: Optional[GcpSqlSettings] = field(default=None)
+    settings: GcpSqlSettings = field(default=None)
```

Comment:
Is this correct? If the field is no longer optional, then the default can't be None.

Reply:
It is still optional, that was just an experimental change. But I rolled it back in https://github.com/someengineering/resoto/pull/1528/commits/ae6308144b5725f05ffa514d93aaffd3b63ede67 🤔